test/rocprim/test_block_load_store.kernels.hpp

```diff
@@ -160,178 +169,369 @@ typedef ::testing::Types<
+                         vector_params<char, rocprim::detail::char4, 8, true>,
+                         vector_params<char, char, 11, false>,
+                         vector_params<char, rocprim::detail::char4, 16, true>,
+
+                         vector_params<short, short, 3, false>,
+                         vector_params<short, rocprim::detail::short4, 4, true>,
+                         vector_params<short, short, 7, false>,
+                         vector_params<short, rocprim::detail::short4, 8, true>,
+                         vector_params<short, short, 11, false>,
+                         vector_params<short, rocprim::detail::short4, 16, true>,
+
+                         vector_params<float, int, 3, false>,
+                         vector_params<float, rocprim::detail::int4, 4, true>,
+                         vector_params<float, int, 7, false>,
+                         vector_params<float, rocprim::detail::int4, 8, true>,
+                         vector_params<float, int, 11, false>,
+                         vector_params<float, rocprim::detail::int4, 16, true>,
+
+                         vector_params<int2, rocprim::detail::int2, 3, false>,
+                         vector_params<int2, rocprim::detail::int4, 4, true>,
+                         vector_params<int2, rocprim::detail::int2, 7, false>,
+                         vector_params<int2, rocprim::detail::int4, 8, true>,
+                         vector_params<int2, rocprim::detail::int2, 11, false>,
+                         vector_params<int2, rocprim::detail::int4, 16, true>,
+
+                         vector_params<float2, rocprim::detail::int2, 3, false>,
+                         vector_params<float2, rocprim::detail::int4, 4, true>,
+                         vector_params<float2, rocprim::detail::int2, 7, false>,
+                         vector_params<float2, rocprim::detail::int4, 8, true>,
+                         vector_params<float2, rocprim::detail::int2, 11, false>,
+                         vector_params<float2, rocprim::detail::int4, 16, true>,
+
+                         vector_params<char4, int, 3, false>,
+                         vector_params<char4, rocprim::detail::int4, 4, true>,
+                         vector_params<char4, int, 7, false>,
+                         vector_params<char4, rocprim::detail::int4, 8, true>,
+                         vector_params<char4, int, 11, false>,
+                         vector_params<char4, rocprim::detail::int4, 16, true>>
+    VectorParams;
+
+template<rocprim::block_load_method  LoadMethod,
+         rocprim::block_store_method StoreMethod,
+         unsigned int                BlockSize>
+struct enable_block_load_store_test
+{
+    static constexpr bool value
+        = (LoadMethod != rocprim::block_load_method::block_load_warp_transpose
+           && StoreMethod != rocprim::block_store_method::block_store_warp_transpose)
+          || BlockSize % ROCPRIM_WAVEFRONT_SIZE == 0;
```

Comment:
This macro does not appear to be defined anywhere.  The PR does not compile

Reply:
Oops, the cherry-pick was dependent on a change that wasn't upstreamed yet. I've changed it in 25de739d1228272ddc3a7958eee58781e6b532bb. Apologies!

Reply:
All good. Thanks!